page/sass/commit_funds.scss

```diff
@@ -0,0 +1,19 @@
+.section.commit-funds{
+    h3{
```

Comment:
I am sure that there are some similarity. If font is bigger/smaller just a little bit then we can assume its the same (mockups are not 100% correct). Lets see later what to do with that.